testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/AbstractAdvancedBrokerTest.java

```diff
@@ -157,7 +156,7 @@ public void testAccountManagementLinkedIdentityAlreadyExists() {
         driver.navigate().to(getAccountUrl(getConsumerRoot(), bc.consumerRealmName()));
         logInWithBroker(bc);
         waitForAccountManagementTitle();
-        accountUpdateProfilePage.assertCurrent();
```

Comment:
This will need to be updated to not rely on the account console at all, but I presume that will be handled in a follow-up?